modules/tinymce/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Notifications no longer require a timeout to disable the close button #TINY-6679
 - The editor theme is now fetched in parallel with the icons, language pack and plugins #TINY-8453
 - Calls to `editor.selection.setRng` now update the cursor position bookmark used when focus is returned to the editor #TINY-8450
+- The default minimum height of editor has been changed to 100px to prevent the UI disappearing while resizing #TINY-6860
```

Comment:
This is the opposite of what I said needed doing... this should be changed and the other should be in Improved.

Reply:
Sorry. I didn't read it carefully. 
However, makes more sense to me for this to be in the Improved section since it improves the usability of the UI.

Reply:
It's not an improvement, if anything it's a bug fix, but because it's changing behaviour the `Changed` is a higher precedence so it should go in there.